contrib/aws/aws-sdk-go-v2/aws/aws.go

```diff
@@ -74,24 +76,25 @@ func (mw *traceMiddleware) startTraceMiddleware(stack *middleware.Stack) error {
 	) (
 		out middleware.InitializeOutput, metadata middleware.Metadata, err error,
 	) {
-		operation := awsmiddleware.GetOperationName(ctx)
-		serviceID := awsmiddleware.GetServiceID(ctx)
+		awsOperation := awsmiddleware.GetOperationName(ctx)
+		awsService := awsmiddleware.GetServiceID(ctx)
```

Comment:
nit: Curious why `operation` needed to be changed to `awsOperation`. Since we're in an AWS package for that contrib, I think `operation` is sufficiently descriptive unless there's another operation this could be in this package.

Reply:
The motivation was because both `service` and `operation` is terminology we use in the context of tracing, I felt confused a few times while going over the code and I decided to change it, but I have no strong opinion so I can change it back if preferred

Reply:
Gotcha. I don't feel very strongly, but I do still think that `service` and `operation` would be sufficient in this context.

Reply:
changed!